src/main/java/se/munchbox/user/UserService.java

```diff
@@ -0,0 +1,29 @@
+package se.munchbox.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service()
```

Comment:
Should there be parenthesis here?

Reply:
No, I belive.

Reply:
Can you specify which line exactly? Is it line 8?

Reply:
Parentheses shouldn't be needed if it will be empty, as far as I know

Reply:
Yes for Service annotation we dont need parantheses, but it doesnt give any error, anyway I will remove it

Reply:
Deleted Parenthesis